packages/payload/src/config/types.ts

```diff
@@ -977,6 +983,48 @@ export type Config = {
   email?: EmailAdapter | Promise<EmailAdapter>
   /** Custom REST endpoints */
   endpoints?: Endpoint[]
+  /**
+   * Options for folder view within the admin panel
+   */
+  folders?: {
```

Comment:
Feels to me like most of this belongs on the collection config, not at the root. E.g. if I wanted to enable folders on a collection, my first instinct would be do so on that collection's config. Otherwise the collection config is fractured into multiple places throughout the greater config.

Reply:
Done 👍